### PR TITLE
fix(extension): remove calc from nft image height and width

### PR DESCRIPTION
**Pull request by jplorek-atix**
# Checklist

- [ ] JIRA - \<[LW-7102](https://input-output.atlassian.net/browse/LW-7102)>
- [ ] Proper tests implemented
- [ ] Screenshots added.

---

## Proposed solution

Removes `calc` from css height and width properties in NFT image component

## Testing

Describe here, how the new implementation can be tested.
Provide link or briefly describe User Acceptance Criteria/Tests that need to be met

## Screenshots

image

[LW-7102]: https://input-output.atlassian.net/browse/LW-7102?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

---
# Allure report
`allure-report-publisher` generated test report!

**smokeTests**: ❌ [test report](https://lace-qa:8PP5wBaDV6UoXj@d1terqjryk7mu9.cloudfront.net/linux/chrome/710/5422062254/index.html) for [ce7fbdf7](https://github.com/input-output-hk/lace/pull/177/commits/ce7fbdf717cbaf1b2414ac2a8479060ee1b3cd6b)
|       | passed | failed | skipped | flaky | total | result |
|-------|--------|--------|---------|-------|-------|--------|
| Total | 34     | 2      | 0       | 0     | 36    | ❌     |

### CODE REVIEW

File: packages/core/src/ui/components/Nft/NftImage.module.scss [hunk 1]
```diff
@@ -1,12 +1,12 @@
 @import '../../styles/theme.scss';
 .nftImage {
   border-radius: inherit;
-  height: calc(100% + 2px);
-  left: -1px;
+  left: 0;
   object-fit: cover;
   position: absolute;
-  top: -1px;
-  width: calc(100% + 2px);
```
Comment (DominikGuzei) [inline packages/core/src/ui/components/Nft/NftImage.module.scss:9]: Why did we have + 2px there?

Comment (jplorek-atix) [reply packages/core/src/ui/components/Nft/NftImage.module.scss:9]: There was a border that we were hiding under the image, that is why we were adding this 2px

**Review by vetalcore — APPROVED** [review]

**Review by milagi — APPROVED** [review]

**Comment by marcin-mazurek** [comment]
Pasting comment from @allanleone
>As we discussed in one of the refinements, If I disabled all visual enhancements of the OS, and without the kerning/smoothness, combined with no hardware acceleration on chrome, any content will look very pixelated (in special high contrast ones like the handle text), and this could be one of possible reasons why borders also may present the mentioned problem – example attached
>
> related to the proportion, the changes to width 100% (removing the calc)( from the branch “nft-image-scale” seems to fix it, but a disclaimer, if any NFT image have a different aspect ratio from 1:1, it will deform the image while forcing to be squared (TBH I don’t know how often a NFT can be non-squared)
>
> if required, one possible alternative would be create a square and adjustable container, and apply the image as background, with the “background-size: cover; and background-position: center center;” – it crops the non-squared images, but keeps the proportion (something similar it’s being used for the parallax effect at the [prototype](https://d1qivfvv04x1it.cloudfront.net/nfts))
>
> so I believe it’s just a matter to evaluate if that’s not necessary, and if “squared shape” covers all, it’s all good to proceed 👍

So, @allanleone, from what I understand, the current implementation (before the fix), assumes that image representing a NFT is square. Let's continue with that assumption for this simple fix, and get it merged. And separately - let's check if that assumption is correct - I'll start a relevant Slack thread. Happy with this approach?

**Comment by marcin-mazurek** [comment]
I've had some time today to verify this further, actually, we seem to be handling non-square NFTs perfectly fine, both in the current version and after @jplorek-atix's fixes, thanks to `object-fit: cover;`. See the screenshots below from @jplorek-atix's branch.

Screenshot 2023-06-30 at 12 28 40
Screenshot 2023-06-30 at 12 28 49
Screenshot 2023-06-30 at 12 33 19
Screenshot 2023-06-30 at 12 28 53

**Comment by allanleone** [comment]
> 
> Regarding disabling the font kerning, hardware acceleration, etc. - (..)

No worries, it was just an observation, as you mentioned, not much can be done when core visual OS settings are turned off, and I agreed with the approach to, let's assume it's squared for now, and to raise this to check/fix on another task 👍

**Comment by marcin-mazurek** [comment]
@bslabiak can you double check the failing smoke tests?

**Comment by bslabiak** [comment]
Failed smokes are not related with this branch changes

**Pull request merged**